Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo

```diff
@@ -0,0 +1,85 @@
+within Buildings.ChillerWSE.BaseClasses.Examples.BaseClasses;
+partial model PartialPlantWithControl
+  "Partial model that tests source plant with temperature control"
+  import Buildings;
```

Comment:
Dr. Hu, @JayHuLBL this model has been deleted in the latest update before I made this pull request. I don't have this model in my current branch. Do you have any idea why it's still in the pull request?